lambdas/indexer/app.py

```diff
@@ -137,3 +147,17 @@ def aggregate_retry(event: chalice.app.SQSEvent):
                     batch_size=1)
 def contribute_retry(event: chalice.app.SQSEvent):
     app.index_controller.contribute(event, retry=True)
+
+
+if (
+    # Determining the bucket name requires AWS access
+    not app.unit_test
+    # We only need one Lambda per AWS account because there is only one GitLab
+    # instance per account
+    and config.deployment_stage == config.shared_deployment_stage
+):
+    @app.on_s3_event(bucket=aws.qualified_bucket_name(config.logs_term),
+                     events=['s3:ObjectCreated:*'],
+                     prefix=config.alb_access_log_path_prefix('gitlab'))
+    def forward_alb_logs(event: chalice.app.S3Event):
+        app.log_controller.forward_logs(event)
```

Comment:
Please add unit test coverage of that method. You can use moto to mock the bucket and a log object.